.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -2,7 +2,7 @@
 name: Feature request
 about: Suggest an idea for this project
 title: "[FEATURE]"
-labels: ''
+labels: feature
```

Comment:
Yeah. FYI, this is pretty much an exact copy of this PR: https://github.com/awslabs/soci-snapshotter/pull/83 generated by Github. New PR because that didn't have DCO.